Google.PowerShell/PubSub/GcpsCmdlet.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Google.Apis.Pubsub.v1;
```

Comment:
We have a tool to automatically insert copyright headers on all files. You might want to rerun that on a separate PR, since it will probably fix up a bunch of other files in the repo.